framework/helpers/BaseArrayHelper.php

```diff
@@ -110,17 +110,14 @@ public static function toArray($object, $properties = [], $recursive = true)
      * You can use [[UnsetArrayValue]] object to unset value from previous array or
      * [[ReplaceArrayValue]] to force replace former value instead of recursive merging.
      * @param array $a array to be merged to
-     * @param array $b array to be merged from. You can specify additional
+     * @param array ...$b array to be merged from. You can specify additional
      * arrays via third argument, fourth argument etc.
      * @return array the merged array (the original arrays are not changed.)
      */
-    public static function merge($a, $b)
+    public static function merge(array $a, array $b = [])
```

Comment:
I understand, I'll correct it.
Are yor sure that BC breaks, please specify exactly how?